diffsync/store/__init__.py

```diff
@@ -72,6 +72,10 @@ def get_by_uids(
         """
         raise NotImplementedError
 
+    def _remove_item(self, modelname: str, uid: str):
```

Comment:
I don't think I like having a private but abstract method. If this is something subclasses are responsible for implementing, it shouldn't be private, IMHO.

Reply:
yes, I remember a previous conversation about this public/private question :)